client/my-sites/customer-home/locations/primary/index.jsx

```diff
@@ -49,6 +50,7 @@ const Primary = ( { checklistMode, cards } ) => {
 								: null,
 						} )
 				) }
+			<PerformanceTrackerStop />
```

Comment:
Rendering this component is a way to signal 'this page is ready'. When this is mounted it will collect metadata and send it to Speedcurve.